flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/TableFunction.java

```diff
@@ -106,36 +103,11 @@ public final void setCollector(Collector<T> collector) {
 	 *
 	 * @return {@link TypeInformation} of result type or <code>null</code> if Flink should determine the type
 	 */
+	@Override
 	public TypeInformation<T> getResultType() {
```

Comment:
Should we remove as it has been defined in `CustomTypeDefinedFunction ` ?

Reply:
TableFunction has generic paradigm T, so we override it return TypeInformation<T> to keep compatible.